src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -94,6 +94,17 @@ public GHTeam getTeamByName(String name) throws IOException {
     }
 
     /**
+     * Finds a team that has the given slug in its {@link GHTeam#getSlug()}
+     */
+    public GHTeam getTeamBySlug(String slug) throws IOException {
+        for (GHTeam t : listTeams()) {
+            if(t.getSlug().equals(slug))
```

Comment:
They accepted the enhancement for selecting teams via `:slug` but with the usually "we make no guarantees" line.  So we may or may not see it as a feature of the API in the future.